docs/coordinates/representations.rst

```diff
@@ -191,3 +205,73 @@ A representation class may also have a ``_unit_representation`` attribute
 probably only meaningful for subclasses of
 `~astropy.coordinates.SphericalRepresentation`, where it is assumed that it
 will be a subclass of `~astropy.coordinates.UnitSphericalRepresentation`.
+
+
+Vector arithmetic
+^^^^^^^^^^^^^^^^^
+
+Representations support basic vector arithmetic, in particular taking the
```

Comment:
I tried to do this with the note at the very end, but now agree this would come too late. I thought a bit about changing the text, but ended up deciding to move the note higher up and make it a proper `.. Note::`, so it stands out in the documentation.